### PR TITLE
Adding subscription functionality to the hub controller

### PR DESCRIPTION
**Pull request by danielrischar**
Adds subscribe, verification, and translation to the hub controller.

### CODE REVIEW

File: node/hubController.js [hunk 1]
```diff
@@ -153,6 +153,73 @@ class HubController {
     }
 
     /**
+     * Subscribe for notifications on all resources composing a platform.  Notifications will be posted to to
+     * the callbackURL.
+     */
+    subscribePlatform(hubId, authInfo, opent2tBlob, callbackUrl) {
```
Comment (arjun-msft) [inline node/hubController.js]: can you add tests? I believe these tests are end to end using a device so that should be ok for subscription as well

**Comment by danielrischar** [comment]
I also moved the auth info for the hubController tests into their own git ignored JSON to prevent accidentally committing passwords and secret keys.  The opent2tblobs and ids are also user-unique data, but they don't expose anything private.

File: node/tests/hubController-testConfig.json [hunk 2]
```diff
@@ -1,15 +1,5 @@
 {
     "hubId":"winkHub",
-    "onboardingInfo" : [
-        { 
-            "username": "",
-            "password": ""
-        },
```
Comment (arjun-msft) [inline node/tests/hubController-testConfig.json]: did you mean to delete this section?

**Comment by arjun-msft** [comment]
:shipit:

**Pull request merged**